packaging/ubuntu-16.04/control

```diff
@@ -62,8 +62,8 @@ Depends: adduser,
          systemd,
          ${misc:Depends},
          ${shlibs:Depends}
-Replaces: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9), snap-confine (<< 2.23), ubuntu-core-launcher (<< 2.22)
-Breaks: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9), snap-confine (<< 2.23), ubuntu-core-launcher (<< 2.22)
+Replaces: ubuntu-snappy (<< 1.9), ubuntu-snappy-cli (<< 1.9), snap-wrap (<< 2.23), ubuntu-core-launcher (<< 2.22)
```

Comment:
Is this intentional? We don't have snap-wrap (ever) as a standalone package and we do want to replace snap-conifne